__tests__/utils/result.utils.test.js

```diff
@@ -0,0 +1,165 @@
+import {
+  STORAGE_RESULTS_BOARD,
+  STORAGE_SCORE_LEVEL_MOOD,
+} from "../../src/constants/constants"
+import { Labels } from "../../src/constants/specificLabels"
+import {
+  convertResultsInStorageToContentTable,
+  getEpdsResultsAndMoodInStorage,
+} from "../../src/utils/result.utils"
+
+describe("Result Utils", () => {
+  describe("getEpdsResultsAndMoodInStorage", () => {
+    beforeAll(() => {
+      global.localStorage = {
+        setItem(key, item) {
+          this.state[key] = item
+        },
+        getItem(key) {
+          return this.state[key]
+        },
+      }
+    })
+
+    test("LocalStorage vide", () => {
```

Comment:
`should return null results when local storage is empty`, qu'est-ce que tu en pense ?